lsif-java/src/main/scala/com/sourcegraph/lsif_java/buildtools/GradleJavaToolchains.scala

```diff
@@ -53,9 +55,20 @@ object GradleJavaToolchains {
   ): GradleJavaToolchains = {
     val scriptPath = tmp.resolve("java-toolchains.gradle")
     val toolchainsPath = tmp.resolve("java-toolchains.txt")
+    val gradleVersionPath = tmp.resolve("gradle-version.txt")
     val taskName = "lsifDetectJavaToolchains"
     val script =
-      s"""|allprojects {
+      s"""|
+          |try {
+          |  java.nio.file.Files.write(
+          |    java.nio.file.Paths.get('$gradleVersionPath'),
+          |    [gradle.gradleVersion],
```

Comment:
Is this some scala feature Im not aware of?

Reply:
Which feature? This is a Groovy script, where you have literal syntax `[a]` for lists.